persons.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  # creating accessors
+  attr_accessor :name, :age, :id
+
+  def initialize(name, age = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] kindly add name as an instance variable, you can use `Random.rand(Random.rand(id numbers range params eg 1-1000)` to generate unique ids